### PR TITLE
Show tool tip on command bar buttons even if they are not icon-only

### PR DESCRIPTION
**Pull request by amreladawy**
This enables showing tooltip on command bar buttons even if they are not icon-only.

## Previous Behavior

Command bar buttons show tooltip only if they are in iconOnly mode. 

## New Behavior

Command bar buttons will show tooltip as long as there is a tooltip host props reagrdless if they are iconOnly or not.

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/73316d6dfb7532d97465dd185957670f0d4d277e'>73316d6dfb7532d97465dd185957670f0d4d277e</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react/src/components/CommandBar/CommandBar.base.tsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ export class CommandBarBase extends React.Component<ICommandBarProps, {}> implem
       onClick: this._onButtonClick(item),
```
Comment (fabricteam) [inline packages/react/src/components/CommandBar/CommandBar.base.tsx:179]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28418&commitId=300ca97e60299bc85a5a1cd1b1d39fcc06cb263d&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Root - RTL.hover.chromium.png|69| Changed|
|CommandBar.Root.hover.chromium.png|69| Changed|
|CommandBar.Text only.hover.chromium.png|69| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/BaseButton-mount_base.html">637</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/BaseButton-mount.html">619</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Breadcrumb-mount_base.html">1655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Breadcrumb-mount.html">1665</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Checkbox-mount_base.html">1703</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Checkbox-mount.html">1693</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/CheckboxBase-mount_base.html">1462</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/CheckboxBase-mount.html">1495</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ChoiceGroup-mount_base.html">2933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ChoiceGroup-mount.html">2950</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ComboBox-mount_base.html">682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ComboBox-mount.html">666</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/CommandBar-mount_base.html">6238</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/CommandBar-mount.html">6984</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ContextualMenu-mount_base.html">13166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ContextualMenu-mount.html">13360</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DefaultButton-mount_base.html">774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DefaultButton-mount.html">744</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRow-mount_base.html">2188</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRow-mount.html">2210</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRowFast-mount_base.html">2166</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRowFast-mount.html">2181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRowNoStyles-mount_base.html">1983</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DetailsRowNoStyles-mount.html">2017</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Dialog-mount_base.html">2679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Dialog-mount.html">2652</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DocumentCardTitle-mount_base.html">214</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/DocumentCardTitle-mount.html">231</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Dropdown-mount_base.html">1931</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Dropdown-mount.html">1979</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/FocusTrapZone-mount_base.html">1078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/FocusTrapZone-mount.html">1105</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/FocusZone-mount_base.html">1062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/FocusZone-mount.html">1082</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-mount_base.html">41344</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-mount.html">41475</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-virtual-rerender_base.html">17843</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-virtual-rerender.html">19853</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedList-virtual-rerender-with-unmount.html">51134</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-mount_base.html">233</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-mount.html">221</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-virtual-rerender_base.html">206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-virtual-rerender.html">220</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">223</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">226</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/IconButton-mount_base.html">1083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/IconButton-mount.html">1084</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Label-mount_base.html">324</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Label-mount.html">337</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Layer-mount_base.html">2713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Layer-mount.html">2701</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Link-mount_base.html">397</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Link-mount.html">401</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/MenuButton-mount_base.html">970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/MenuButton-mount.html">928</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/MessageBar-mount_base.html">21108</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/MessageBar-mount.html">21308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Nav-mount_base.html">1965</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Nav-mount.html">1948</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/OverflowSet-mount_base.html">786</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/OverflowSet-mount.html">771</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Panel-mount_base.html">1795</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Panel-mount.html">1779</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Persona-mount_base.html">736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Persona-mount.html">747</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Pivot-mount_base.html">872</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Pivot-mount.html">874</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/PrimaryButton-mount_base.html">859</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/PrimaryButton-mount.html">842</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Rating-mount_base.html">4623</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Rating-mount.html">4659</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SearchBox-mount_base.html">929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SearchBox-mount.html">938</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Shimmer-mount_base.html">1901</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Shimmer-mount.html">1923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Slider-mount_base.html">1338</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Slider-mount.html">1363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SpinButton-mount_base.html">2914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SpinButton-mount.html">2923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Spinner-mount_base.html">392</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Spinner-mount.html">381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SplitButton-mount_base.html">1801</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SplitButton-mount.html">1863</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Stack-mount_base.html">428</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Stack-mount.html">414</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/StackWithIntrinsicChildren-mount_base.html">852</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/StackWithIntrinsicChildren-mount.html">848</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/StackWithTextChildren-mount_base.html">2635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/StackWithTextChildren-mount.html">2657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SwatchColorPicker-mount_base.html">6206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/SwatchColorPicker-mount.html">6119</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/TagPicker-mount_base.html">1461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/TagPicker-mount.html">1421</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Text-mount_base.html">363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Text-mount.html">380</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/TextField-mount_base.html">935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/TextField-mount.html">946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-mount_base.html">833</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-mount.html">833</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-virtual-rerender_base.html">590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-virtual-rerender.html">582</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1264</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Toggle-mount_base.html">633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/Toggle-mount.html">610</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/buttonNative-mount_base.html">192</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28418/perf-test/buttonNative-mount.html">194</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react/src/components/CommandBar/CommandBar.base.tsx (hunk 1 above)
Comment (fabricteam) [inline packages/react/src/components/CommandBar/CommandBar.base.tsx:179]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28418&commitId=e139c4a27f52492093f55156ddd4d72e7207753f&env=prod&clientType=FLUENTUI)


  <details><summary><b>CommandBar</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Text only.hover.chromium.png|69| Changed|
|CommandBar.Root.hover.chromium.png|69| Changed|
|CommandBar.Root - RTL.hover.chromium.png|69| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by behowell — CHANGES_REQUESTED** [review]
Hi @amreladawy, thank you for the PR! Unfortunately, as it is, it looks like it's changing the behavior of existing toolbars. The visual regression test shows additional tooltips that shouldn't be there:
image

CommandBar intentionally doesn't show tooltips for buttons with text, as that is redundant information. So it seems to be working as designed.

Could you give some more details about your scenario? Why are you showing tooltips for command bar buttons with labels?

**Comment by behowell** [comment]
I think you can achieve what you want without requiring a change to the core CommandBar control.

If you'd like to have your command bar always display a tooltip when `tooltipHostProps` is specified, you can implement that yourself using a custom button component and the `buttonAs` prop:

```tsx
const CommandBarButtonWithTooltip: React.FunctionComponent<IButtonProps> = props => {
  const { tooltipHostProps, ...buttonProps } = props;

  const button = <CommandBarButton {...buttonProps} />;
  
  return tooltipHostProps ? <TooltipHost {...tooltipHostProps}>{button}</TooltipHost> : button;
};

<CommandBar buttonAs={CommandBarButtonWithTooltip} ... />
```

See an example here: https://codepen.io/behowell/pen/poQKKPj?editors=0011

**Comment by amreladawy** [comment]
Thanks a lot, Ben! Truly appreciated. closing this PR.

**Comment by amreladawy** [comment]
Thanks a lot, Ben! truely appreciated. closing this PR.

**Pull request closed without merging**